pkg/api/adjustment.go

```diff
@@ -129,6 +129,12 @@ func (a *ContainerAdjustment) RemoveDevice(path string) {
 	})
 }
 
+// AddCDIDevice records the addition of the given CDI device to a container.
+func (a *ContainerAdjustment) AddCDIDevice(d *CDIDevice) {
+	a.initLinux()
```

Comment:
I think this can be removed since `CDIDevices` is no longer in the `linux` struct.

Reply:
True and thanks for spotting that. Removed.